build-configs.yaml

```diff
@@ -768,6 +776,13 @@ build_configs:
           arm64: *arm64_defconfig
           x86_64: *x86_64_defconfig
 
+      clang-12:
```

Comment:
If we want to build with the latest clang, we should probably have `clang-latest` instead or just `clang`.  That way, we know when we actually start using clang 12 when it's released.